lib/aruba/platform.rb

```diff
@@ -182,19 +183,16 @@ def simple_table(hash)
     # rubocop:disable Metrics/MethodLength
     # rubocop:disable Metrics/CyclomaticComplexity
     def which(program, path = ENV['PATH'])
-      on_windows = false
-      on_windows = true if File::ALT_SEPARATOR
-
       program = program.to_s
```

Comment:
Nice idea to have lots of little objects. Doesn't all need to go into one PR though.